### PR TITLE
AQQU support

### PR DESCRIPTION
**Pull request by larshp**
test repository = https://github.com/abapGit-tests/AQQU

### CODE REVIEW

File: src/objects/zcl_abapgit_object_aqqu.clas.abap [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    set_default_transport( iv_transport ).
+
+    get_generic( )->delete( iv_package ).
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~deserialize.
+
+    set_default_transport( iv_transport ).
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_aqqu.clas.abap:64]: 😉

File: src/objects/zcl_abapgit_object_aqqu.clas.abap [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+  METHOD zif_abapgit_object~changed_by.
+    rv_user = zcl_abapgit_objects_super=>c_user_unknown.
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~delete.
+
+    set_default_transport( iv_transport ).
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_aqqu.clas.abap:55]: 😉

File: src/objects/zcl_abapgit_object_aqqu.clas.abap (hunk 1 above)
Comment (larshp) [reply src/objects/zcl_abapgit_object_aqqu.clas.abap:64]: argh :) I copied the old code

**Review by mbtools — APPROVED** [review]
👍

**Comment by larshp** [comment]
set_default_transport not fixed yet...

**Comment by larshp** [comment]
ready

**Review by mbtools — APPROVED** [review]
I will add the jump to these classes later

**Pull request merged**